modules/ppcp-settings/src/Endpoint/CommonRestEndpoint.php

```diff
@@ -60,6 +60,20 @@ class CommonRestEndpoint extends RestEndpoint {
 		),
 	);
 
+	/**
+	 * Map the internal flags to JS names.
+	 *
+	 * @var array
+	 */
+	private array $flag_map = array(
+		'country'  => array(
+			'js_name' => 'country',
+		),
+		'currency' => array(
+			'js_name' => 'currency',
```

Comment:
Instead of renaming the variables in the hook, we can directly use the js_name `storeCountry` and `storeCurrency` here and change the hook to 

```js
export const useWooSettings = () => {
	const { flags } = useHooks();
	return flags;
}
```